cypress.json

```diff
@@ -1,17 +1,18 @@
 {
-    "chromeWebSecurity": false,
-    "baseUrl": "http://localhost:3007",
-    "ignoreTestFiles": "**/!(*.spec).ts",
-    "fixturesFolder": "integration_tests/fixtures",
-    "integrationFolder": "integration_tests/integration",
-    "pluginsFile": "integration_tests/plugins/index.ts",
-    "screenshotsFolder": "integration_tests/screenshots",
-    "supportFile": "integration_tests/support/index.ts",
-    "videosFolder": "integration_tests/videos",
-    "reporter": "cypress-multi-reporters",
-    "reporterOptions": {
-      "configFile": "reporter-config.json"
-    },
-    "videoUploadOnPasses": false,
-    "taskTimeout": 60000
+  "chromeWebSecurity": false,
+  "baseUrl": "http://localhost:3007",
+  "ignoreTestFiles": "**/!(*.spec).ts",
+  "fixturesFolder": "integration_tests/fixtures",
+  "integrationFolder": "integration_tests/integration",
+  "pluginsFile": "integration_tests/plugins/index.ts",
+  "screenshotsFolder": "integration_tests/screenshots",
+  "supportFile": "integration_tests/support/index.ts",
+  "videosFolder": "integration_tests/videos",
+  "reporter": "cypress-multi-reporters",
+  "reporterOptions": {
+    "configFile": "reporter-config.json"
+  },
+  "videoUploadOnPasses": false,
+  "taskTimeout": 60000,
+  "downloadsFolder": "integration_tests/downloads"
```

Comment:
I added this line (to tell cypress where to put the downloads), but for some reason my IDE decided to reformat the rest of the file! 🤷‍♂️